### PR TITLE
Jekyll Speedup

### PR DESCRIPTION
**Pull request by Pandapip1**
I can confirm that all these changes do not break the Jekyll website. I don't think there's anything here that will cause undue annoyance, but marking this as a draft and tagging @MicahZoltu just in case...

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -70,3 +71,6 @@ include:
   - LICENSE
 
 markdown_ext: "markdown,mkdown,mkdn,mkd,md"
+
+plugins:
+  - jekyll-remote-theme
```
Comment (Pandapip1) [inline _config.yml:76]:
```suggestion
  - jekyll-remote-theme

```

Comment (Pandapip1) [reply _config.yml]: This has got to be one of the laziest things I've done lol

File: _config.yml [hunk 2]
```diff
@@ -33,21 +33,22 @@ header_pages:
...
-  syntax_highlighter: rouge
+incremental: true
+commonmark:
+  options:
+    - SMART
+    - STRIKETHROUGH_DOUBLE_TILDE
+    - HARDBREAKS
+  extensions:
+    - table  # Some EIPs use this syntax for tables
+    - strikethrough  # Strikethroughs are supported like ~~so~~.
```
Comment (MicahZoltu) [inline _config.yml:46]: I feel like people shouldn't be including ~~strikethrough~~ in EIPs.

File: Gemfile [hunk 3]
```diff
@@ -10,10 +10,15 @@ source "https://rubygems.org"
...
 
+# Use better liquid gem
+gem "liquid-c"
+
 # If you have any plugins, put them here!
 group :jekyll_plugins do
   gem "jekyll-feed", "~> 0.11"
   gem "github-pages", "206"
+  gem "jekyll-commonmark-ghpages"  # Additional performance improvements
+  gem "jekyll-include-cache"
```
[the remark this answers is not in the document]
Comment (Pandapip1) [reply Gemfile:21]: The lockfile does pin the dependencies. The lockfile is also the source of my annoyances, as you can see in the error message.

**Comment by Pandapip1** [comment]
Well, I guess it is speeding up the CI, by making it fail almost immediately 😑

**Comment by Pandapip1** [comment]
Oh I'm stupid. I am so stupid.

I forgot to re-build. 🤦‍♂️

**Comment by Pandapip1** [comment]
I give up.

**Pull request closed without merging**